include/boost/gil/extension/io/bmp/detail/is_allowed.hpp

```diff
@@ -89,4 +89,4 @@ bool is_allowed( const image_read_info< bmp_tag >& /* info */
 } // namespace gil
 } // namespace boost
 
-#endif // BOOST_GIL_EXTENSION_IO_BMP_IO_IS_ALLOWED_HPP
+#endif
```

Comment:
A side comment: do we care about EOL at the EOF ?

Reply:
I think so. Why do you ask ? (I'm typically editing using emacs, and that editor automatically inserts missing EOL at the end.)

Reply:
I think this and (some) other headers are missing the EOL at the end.

Reply:
I'm sorry, I must be too eager trusting how GitHub renders file.